Source/Fuse.Reactive.JavaScript/RootableScriptModule.uno

```diff
@@ -31,10 +31,18 @@ namespace Fuse.Reactive
 			_classInstance.EnsureRooted();
 		}
 
+		internal Dictionary<string, object> Dependencies;
+
 		protected override string GenerateArgs(Context c, ModuleResult result, List<object> args)
 		{
 			var argsString = base.GenerateArgs(c, result, args);
 
+			foreach (var dep in Dependencies) 
+			{
+				argsString += ", " + dep.Key;
```

Comment:
Stricly speaking yes, but there are tons of cases like this already and should be handled more generally, not in this site specifically.

The behavior of the current implementation will be acceptable imho, you will get an exception from trying to compile the script as expected. The error message might not be the friendliest, that's all